src/components/SelectedTableNotification.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC } from "react";
+import { Button, Notification } from "@canonical/react-components";
+
+interface Props {
+  totalCount: number;
+  filteredNames: string[];
+  itemName: string;
+  parentName: string;
+  selectedNames: string[];
+  setSelectedNames: (val: string[]) => void;
+}
+
+const SelectedTableNotification: FC<Props> = ({
+  totalCount,
+  filteredNames,
+  itemName,
+  parentName,
+  selectedNames,
+  setSelectedNames,
+}: Props) => {
+  const isAllSelected =
+    selectedNames.length === filteredNames.length && selectedNames.length > 0;
+
+  const selectAll = () => {
+    setSelectedNames(filteredNames);
+  };
+
+  const selectNone = () => {
+    setSelectedNames([]);
+  };
+
+  return (
+    <Notification borderless className="u-no-margin--bottom" title="Selection">
```

Comment:
I think that we said the first row (the tile) is superfluous. I think it should be removed as it occupies one row of precious space.
```suggestion
    <Notification borderless className="u-no-margin--bottom">
```